orchard-core/src/main/resources/reference.conf

```diff
@@ -22,10 +22,21 @@ com.salesforce.mce.orchard {
     }
 
    activity {
-       checkProgressDelay = 10 seconds
+       checkProgressDelay {
+           type = "JitteredDelay"
+           params = {
+               minDelay = 90 seconds
+               maxDelay = 150 seconds
+           }
```

Comment:
How easy is it to simplify it to something like this:
```
checkProgressDelay {
  jitteredDelay {
    minDelay {
      minDelay = 90 seconds
      maxDelay = 150 seconds
    }
  }
}
```

Reply:
and also let's make sure `checkProgressDelay = 10 seconds` is the same as configuring it with type `FixedDelay`, so that the configuration is backwards compatible